packages/fxa-payments-server/package.json

```diff
@@ -11,7 +11,9 @@
     "build": "react-scripts build",
     "eject": "react-scripts eject",
     "test": "npm-run-all test:*",
-    "test:ts": "cross-env TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\", \"jsx\": \"react\"}' nyc --reporter=html --reporter=text mocha -r ts-node/register -r @babel/register -r jsdom-global/register -r test-setup.ts src/**/*-test.{ts,tsx} --recursive --timeout 5000"
+    "test:ts": "cross-env TS_NODE_COMPILER_OPTIONS='{\"module\":\"commonjs\", \"jsx\": \"react\"}' nyc --reporter=html --reporter=text mocha -r ts-node/register -r @babel/register -r jsdom-global/register -r test-setup.ts src/**/*-test.{ts,tsx} --recursive --timeout 5000",
+    "storybook": "start-storybook -p 6006",
+    "build-storybook": "build-storybook"
```

Comment:
I wasn't able to run `npm run build-storybook` due to some TypeScript errors, like:

```
ERR! => Failed to build the preview                                           0 ERR! /Users/ianbicking/src/fxa/packages/fxa-payments-server/src/components/Profile.stories.tsx
ERR! TypeScript error in /Users/ianbicking/src/fxa/packages/fxa-payments-server/src/components/Profile.stories.tsx(10,7):
ERR! Type '{ avatar: string; displayName: string; email: string; }' is missing the following properties from type 'Profile': amrValues, avatarDefault, locale, twoFactorAuthentication, uid  TS2739
ERR! 
ERR!      8 |       error: null,
ERR!      9 |       loading: false,
ERR!   > 10 |       result: {
ERR!        |       ^
ERR!     11 |         avatar: '',
ERR!     12 |         displayName: 'Foo Barson',
ERR!     13 |         email: 'foo@example.com',
ERR! /Users/ianbicking/src/fxa/packages/fxa-payments-server/src/routes/Product.stories.tsx
ERR! TypeScript error in /Users/ianbicking/src/fxa/packages/fxa-payments-server/src/routes/Product.stories.tsx(45,5):
```

Reply:
D'oh - looks like some code that I split the wrong way when I untangled that PR from earlier. Will update with fixes